packages/eas-cli/src/project/remoteVersionSource.ts

```diff
@@ -102,3 +127,62 @@ export function getBuildVersionName(platform: Platform): string {
     return 'buildNumber';
   }
 }
+
+export async function ensureAppVersionSourceIsSetAsync(
+  easJsonAccessor: EasJsonAccessor
+): Promise<AppVersionSourceUpdateOption | undefined> {
+  Log.log(
+    `This project is using the default app version source, which currently is the "remote" version source. Prior to version 10.0.0 of CLI the default used to be the "local" version source. App version source can be set for you automatically, or you can configure it yourself - if you wish to use the default "remote" version source add ${chalk.bold(
```

Comment:
I like it a lot, I think this is the clearest option! Could I suggest a small wording change? I think it would help to briefly explain what the version source means.


**Please select your app version source. With "local" the build android.versionCode / ios.buildNumber needs to be incremented manually in app.json / app.config.js, and with "remote" this value is stored in EAS and will be incremented automatically with each build. Until now, this project has been using the "local" version source (which was previously the default). App version source can be set for you automatically, or you can configure it yourself - if you wish to use the default "remote" version source add**

Reply:
I updated the message, thank you for the feedback 🙇